sqlflow/client.py

```diff
@@ -7,67 +7,73 @@
 import sqlflow.proto.sqlflow_pb2 as pb
 import sqlflow.proto.sqlflow_pb2_grpc as pb_grpc
 
-_LOGGER = logging.getLogger(__name__)
+logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
```

Comment:
It does not suggest to config this in library code

Reply:
Changed to only set `_LOGGER`'s level and handler, instead of global config.